flowpm/tfbackground.py

```diff
@@ -609,6 +644,8 @@ def maybe_compute_ODE(cosmo, log10_amin=-2, steps=256):
     # If cache is found in the cosmo dictionary it means the ODE has already
     # been computed
     cache = cosmo._workspace['cache_ODE']
+    # Checking that the stored ODE results have the right lenght
+    assert len(cache['a']) == steps
```

Comment:
hummm did you add that back? I think the assert was causing some issues inside tf.functions so w got rid of it